idl/github.com/uber/cadence/history.thrift

```diff
@@ -322,6 +327,18 @@ service HistoryService {
       4: ShardOwnershipLostError shardOwnershipLostError,
     )
 
+  /**
+  * DeleteWorkflowExecutionSignal is used to delete a signal request ID that was previously recorded.  This is currently
+  * used to clean execution info when signal decision finished.
+  **/
+  void DeleteWorkflowExecutionSignal(1: DeleteWorkflowExecutionSignalRequest deleteRequest)
```

Comment:
Let's rename this to RemoveSignalMutableState

Reply:
Done